content/blog/2024/delivery-improvements/index.md

```diff
@@ -0,0 +1,119 @@
+
+**Strategic goals and major projects**: Next we define strategic goals that describe the most important progress we must make as an organization.
+This considers our current capabilities and challenges, as well as the major projects that we're already committed to (like grants).
+Our initiatives (described below) should each represent progress towards one or more of our goals and major projects.
+
+**Strategic initiatives**: These are major thrusts of work that represent progress towards our goals.
+They range in time from weeks to months, and generally require coordination and action across each of 2i2c's functional areas.
+Initiatives exist in a dedicated board, where we shape and scope them with enough information to understand them and prioritize.
+Once an initiative is in progress, it begins driving work on a two-week planning cycle.
+All in-progress initiatives should make up a significant percentage of 2i2c's total work allocation.
+
+**Operational boards**: Operational boards are used to track our day-to-day workstreams.
+We organize into two-week sprints, with a collection of work pulled into each sprint according to in-progress initiatives and the other types of reactive and operational work on our plate (for example, responding to support tickets is reactive work).
+Within each initiative, we coordinate across our teams in order to understand the next steps needed and who is responsible for doing it.
+This helps teams plan the work for their next iteration.
+
+This system is very much still a work in progress, and we've already identified a number of ways that we'd like to improve it moving forward.
+For example, we'd like to find more efficient ways of coordinating across our team, and encouraging team members to own their own work.
+
+### Changes to our team culture
+
+In addition to our system around planning work, we've also identified a number of ways that we can improve our team's culture related to work.
+As a distributed organization, one of our biggest challenges has been the additional friction that comes with communicating across many time zones.
+It's much harder to solve a problem or get help when responses come with long gaps of time.
+
+As a result, we've defined several practices that will help our team members grow their autonomy and independence, while still moving towards the same targets and goals.
+This is ongoing work that we aim to continue developing over the next few months.
+
+**Personal growth**. 
+First, we've started an audit of 2i2c's organizational roles and the responsibilities and the pathways for growth with each of them.
+This began with our engineering team, but will continue with other areas of the organization as well.
+We believe that defining personal growth trajectories will help identify the gaps in our team's skills that make it more difficult to act efficiently and independently.
```

Comment:
Could we change "personal growth" to "professional growth" or "professional development"? I'd like to highlight that the growth is focused on people's professional skills and qualities.